### ISSUE TITLE
Export and import per activity type

### ISSUE THREAD

**Issue by nixbits** [OpenTracksApp/OpenTracks#1659]
Awesome app. I paid for it on google play after f-droid testing.

**Feature request:**
Ability to import and export tracks per activity type and not all recorded tracks. 
Currently the options are only import all and or export all.

For example, lets say we just want to export bicycle tracks and import them on another phone profile/account or a new phone.

**Comment by dr-m** [comment]
Indeed, this app is awesome, many improvements in the past few years. Last year I was still using [AAT](https://f-droid.org/de/packages/ch.bailu.aat/).

But, the data export/import could be a little easier. Sorry, in the following I am trying to translate the UI terms from my local language into English.

Currently, the export is:
* Hidden in "settings" and "import/export" as "export all".
* Not asking for the format each time when exporting or sharing; it can only be chosen in "settings" under "import/export".

I think that the following could be helpful:
* Implement "filtering" or "selection by condition" in the track list window, instead of only having it under "aggregate statistics" after selecting one or more recorded tracks.
* Offer more filtering criteria (e.g., recorded with/without heart rate sensor).
* Offer user-defined persistent filters, such as "mountain bicycling in 2024", "road bicycling in 2024", "jogging in 2023".
* In the track list context menu, allow the selected or filtered tracks to be exported or shared in the user-specified format.
